Automattic/newspack-block-theme
# feat: add comments sidebar

### All Submissions:






### Changes proposed in this Pull Request:

This PR moves the comments functionality to a hidden sidebar, which can be toggled by a button within the content. While far from perfect, this is a good starting point. Ideally, we would want the button to display only when comments are enabled - perhaps this can be achieved with some JS? cc @laurelfulford.

![comments-demo](https://github.com/Automattic/newspack-block-theme/assets/177929/bce29555-b20f-45f3-910c-2bb7ca029fd7)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check a post
3. Try to open the comments

### Other information:
